### PR TITLE
Refactor of Liqo controllers constants

### PR DESCRIPTION
**Pull request by Andreagit97**
# Description

This PR creates a new folder for Liqo controllers constants (from liqo/pkg/liqo-controller-manager to liqo/pkg/consts). Constants will be divided in files, which will specify their context.

### CODE REVIEW

File: pkg/consts/namespaceMapping.go [hunk 1]
```diff
@@ -1,9 +1,10 @@
-package liqo_controller_manager
+package consts
 
 const (
 	VirtualNodeClusterId            = "cluster-id" //"remote.liqo.io/clusterId"
```
Comment (palexster) [inline pkg/consts/namespaceMapping.go:4]: You need a space after the //

**Review by palexster — APPROVED** [review]

File: pkg/consts/namespaceMapping.go (hunk 1 above)
Comment (Andreagit97) [reply pkg/consts/namespaceMapping.go:4]: Thank you.

**Review by aleoli — APPROVED** [review]

**Pull request merged**